team-validator.js

```diff
@@ -202,6 +178,19 @@ if (!process.send) {
 	});
 }
 
+/**
+ * Validates a username or Pokemon nickname
+ */
+
+var bannedNameStartChars = {'~':1, '&':1, '@':1, '%':1, '+':1, '-':1, '!':1, '?':1, '#':1, ' ':1};
+function getName (name) {
+	name = Tools.getName(name);
+	while (bannedNameStartChars[name.charAt(0)]) {
```

Comment:
We might as well do `name.charAt(0) in bannedNameStartChars`.

Wait, why is this here? This should be in `Tools.getName`...